source/_components/sensor.bbox.markdown

```diff
@@ -34,11 +34,18 @@ sensor:
       - current_up_bandwidth
 ```
 
-Configuration variables:
-
-- **monitored_variables** array (*Required*): Sensors to display in the frontend.
-  - **down_max_bandwidth**: Maximum bandwidth available for download.
-  - **up_max_bandwidth**: Maximum bandwidth available for upload.
-  - **current_down_bandwidth**: Instant measure of the current used bandwidth for download.
-  - **current_up_bandwidth**: Instant measure of the current used bandwidth for upload.
-
+{% configuration %}
```

Comment:
Pherhaps you could also add the variable `name`, [see more](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/bbox.py#L47)

Reply:
Well spotted @klaasnicolaas thanks

Reply:
👍 No problem 😄